modules/redpanda/options.go

```diff
@@ -155,3 +160,13 @@ func WithListener(lis string) Option {
 		})
 	}
 }
+
+// WithBootstrapConfig adds an arbitrary config kvp to the Redpanda container.
```

Comment:
question: This seems like it might get cumbersome to specify multiple values, is there a reason we don't just allow a string to include, which would support arbitrary levels of yaml too?

Reply:
I like having a map because readers will identify the key-value pairs very easily. Of course you can do it with a multiline string, but I see using a map more "typed". OTOH if you all agree the string is fine, I'm fine too

Reply:
The reason I asked was yaml is typically structured, so not all values are at the base level like the example, [here's full config example](https://docs.redpanda.com/current/reference/console/config/) so if if none route elements might be needed this approach would be limited.

Someone who knows redpanda should comment if structured config for bootstrap could be needed?

Reply:
Our cluster config and thus bootstrap yaml file is always a flat key value pair - nesting is not allowed

Reply:
Thanks for confirming